### ISSUE TITLE
react-hot-loader@v3 breaks babel-plugins 

### ISSUE THREAD

**Issue by Always-Sunny** [gaearon/react-hot-loader#660]
Upgrading to   "react-hot-loader": "v3.0.0" is causing conflict with other babel-plugins.

It was working fine until I installed babel-preset-stage-2 and added it to my .babelrc

`npm install --save-dev babel-preset-stage-2`

&&

```
{
  "presets": ["env", "react", "stage-2"],
  "plugins": ["react-hot-loader/babel"]
}
```
After I added "stage-2" to my presets and tried to start my server, I received this error: 

> ReferenceError: Unknown plugin "react-hot-loader/babel"

Downgrading react-hot-loader back to v3.0.0-beta.7 fixed the issue and now I can use the stage-2 preset along with react-hot-loader. 

I also tried installing babel-plugin-transform-class-properties and adding it to my plugins (in stead of the babel preset) and received the same error

**Comment by drcmda** [comment]
Not sure if related but updating to 3.0.0 from beta7 crashes my app as well. I have tried to track it down and it seems to complain about one of my classes, but it's hard to tell what's going on. Going back to beta7 or removing react-hot-loader immediately makes it work again.

<img width="850" alt="screen shot 2017-10-11 at 09 59 05" src="https://user-images.githubusercontent.com/2223602/31428397-e0e3607e-ae6a-11e7-9bed-9c4c01276a62.png">

**Comment by neoziro** [comment]
@drcmda it is an other bug related to #464, can you please fill a new issue?

**Comment by neoziro** [comment]
@drcmda will do it

**Comment by drcmda** [comment]
@neoziro saw it too late, thanks for everything!

**Comment by neoziro** [comment]
Closed as duplicate of #662, by the way it is fixed in [next](https://github.com/gaearon/react-hot-loader/tree/next).
